### DISCUSSION TITLE
Default Maui Blazor Project doesn't run out of the box?????????

### DISCUSSION THREAD

**Discussion by gautambjain** [dotnet/maui#19146] [Q&A]
This is crazy. I have the latest VS 17.8.2 and .Net 8. Upgraded yesterday.

Created the default MAUI Blazor project but gives below error:

Severity	Code	Description	Project	File	Line	Suppression State
Error		DEP0700: Registration of the app failed. [0x80073CF6] AppxManifest.xml(34,27): error 0x80070002: Cannot install or update package com.companyname.xxxxxxxx.mauiblazor_9zz4h110yvjzm because the splash screen image [splashSplashScreen.png] cannot be located. Verify that the package contains an image that can be used as a splash screen for the application, and that the package manifest points to the correct location in the package where this splash screen image can be found.

All I can see in Package.appxmanifest is $ placeholder $ everywhere.

Having been doing Xamarin Forms since 2013. But looks like MAUI is going to be a bad experience from what others are saying. But some few talked positive about MAUI Blazor though. So tried one. Failed on the first step. This itself is intolerable. If the start is so bad how will the journey be?

**Comment by SpaceWalkerCZ** [comment]
Dude, it's so frustrating. Dependencies just don't work RN... I tought I was going crazy.
